<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>青马终结者 v0.5 by Zero</string>
  </property>
  <widget class="QLabel" name="lbl_id">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>神秘链接:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_url">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pte_message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next_question">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>72</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一题</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_answer">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_answer">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_unit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string extracomment="2">形势与政策</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="7">毛概</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="8">思修</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="9">马原</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="10">近代史</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="12">学习新思想</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="15">社会主义先进文化</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="16">不忘初心牢记使命</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="19">新中国成立70周年</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="47">新型冠状病毒防治</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="48">十九届四中全会</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="-1">错题学习</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_change_unit">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择科目:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_change_font_size">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>40</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选择字号:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_change_font_size">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20px</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_ps">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PS:换题是不会记录错误的哦~</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_answeer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>你的答案：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cheat">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>400</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>作弊</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_right">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>对：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_total">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>40</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>总：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_right_counter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_total_counter">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>正确率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rate">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>41</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00%</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QM_Terminator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_submit</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>btn_submit_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_next_question</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>btn_next_question_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_answer</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>btn_answer_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_change_font_size</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainForm</receiver>
   <slot>cb_change_font_size(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_answer</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainForm</receiver>
   <slot>le_answer_input(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_unit</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainForm</receiver>
   <slot>cb_unit_change(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cheat</sender>
   <signal>clicked()</signal>
   <receiver>MainForm</receiver>
   <slot>btn_cheat_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_submit_click()</slot>
  <slot>btn_next_question_click()</slot>
  <slot>btn_answer_click()</slot>
  <slot>cb_unit_change(int)</slot>
  <slot>cb_change_font_size(QString)</slot>
  <slot>le_answer_input(QString)</slot>
  <slot>btn_cheat_click()</slot>
 </slots>
</ui>
